### DISCUSSION TITLE
media remove does not work in cronjob?

### DISCUSSION THREAD

**Discussion by Corrosive1** [mastodon/mastodon#31602] [Q&A]
Hello, i have a file called mediaremove.sh with the following content >>

```
RAILS_ENV=production /home/mastodon/live/bin/tootctl media remove --days 4;
RAILS_ENV=production /home/mastodon/live/bin/tootctl accounts prune;
RAILS_ENV=production /home/mastodon/live/bin/tootctl media remove-orphans;
RAILS_ENV=production /home/mastodon/live/bin/tootctl media remove --remove-headers --include-follows --days 4;
```

i setup a cronjob as user "mastodon" with either of the following 2 i tried 

cd /home/mastodon/live; ./mediaremove.sh;
/home/mastodon/live/mediaremove.sh;

when i run ./mediaremove.sh from putty it works perfectly, when i run it as cronjob i only get this >>

`/usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:214:in `rescue in load_spec_files': https://github.com/stanhu/omniauth-cas.git (at 4211e6d@4211e6d) is not yet checked out. Run `bundle install` first. (Bundler::GitError)
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:210:in `load_spec_files'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/path.rb:107:in `local_specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:178:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/lazy_specification.rb:88:in `__materialize__'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:75:in `block in materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `map!'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:468:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:190:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:238:in `specs_for'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/runtime.rb:18:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler.rb:151:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `block in <top (required)>'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:136:in `with_level'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:88:in `silence'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `<top (required)>'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /home/mastodon/live/config/boot.rb:10:in `<top (required)>'
	from /home/mastodon/live/bin/tootctl:4:in `require_relative'
	from /home/mastodon/live/bin/tootctl:4:in `<main>'
/usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/path.rb:209:in `load_spec_files': The path `/home/mastodon/live/vendor/bundle/ruby/3.0.0/bundler/gems/omniauth-cas-4211e6d05941` does not exist. (Bundler::PathError)
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:211:in `load_spec_files'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/path.rb:107:in `local_specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:178:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/lazy_specification.rb:88:in `__materialize__'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:75:in `block in materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `map!'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:468:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:190:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:238:in `specs_for'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/runtime.rb:18:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler.rb:151:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `block in <top (required)>'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:136:in `with_level'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:88:in `silence'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `<top (required)>'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /home/mastodon/live/config/boot.rb:10:in `<top (required)>'
	from /home/mastodon/live/bin/tootctl:4:in `require_relative'
	from /home/mastodon/live/bin/tootctl:4:in `<main>'
/usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:214:in `rescue in load_spec_files': https://github.com/stanhu/omniauth-cas.git (at 4211e6d@4211e6d) is not yet checked out. Run `bundle install` first. (Bundler::GitError)
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:210:in `load_spec_files'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/path.rb:107:in `local_specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:178:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/lazy_specification.rb:88:in `__materialize__'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:75:in `block in materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `map!'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:468:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:190:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:238:in `specs_for'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/runtime.rb:18:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler.rb:151:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `block in <top (required)>'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:136:in `with_level'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:88:in `silence'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `<top (required)>'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /home/mastodon/live/config/boot.rb:10:in `<top (required)>'
	from /home/mastodon/live/bin/tootctl:4:in `require_relative'
	from /home/mastodon/live/bin/tootctl:4:in `<main>'
/usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/path.rb:209:in `load_spec_files': The path `/home/mastodon/live/vendor/bundle/ruby/3.0.0/bundler/gems/omniauth-cas-4211e6d05941` does not exist. (Bundler::PathError)
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:211:in `load_spec_files'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/path.rb:107:in `local_specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:178:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/lazy_specification.rb:88:in `__materialize__'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:75:in `block in materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `map!'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:468:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:190:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:238:in `specs_for'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/runtime.rb:18:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler.rb:151:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `block in <top (required)>'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:136:in `with_level'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:88:in `silence'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `<top (required)>'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /home/mastodon/live/config/boot.rb:10:in `<top (required)>'
	from /home/mastodon/live/bin/tootctl:4:in `require_relative'
	from /home/mastodon/live/bin/tootctl:4:in `<main>'
/usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:214:in `rescue in load_spec_files': https://github.com/stanhu/omniauth-cas.git (at 4211e6d@4211e6d) is not yet checked out. Run `bundle install` first. (Bundler::GitError)
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:210:in `load_spec_files'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/path.rb:107:in `local_specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:178:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/lazy_specification.rb:88:in `__materialize__'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:75:in `block in materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `map!'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:468:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:190:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:238:in `specs_for'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/runtime.rb:18:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler.rb:151:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `block in <top (required)>'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:136:in `with_level'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:88:in `silence'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `<top (required)>'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /home/mastodon/live/config/boot.rb:10:in `<top (required)>'
	from /home/mastodon/live/bin/tootctl:4:in `require_relative'
	from /home/mastodon/live/bin/tootctl:4:in `<main>'
/usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/path.rb:209:in `load_spec_files': The path `/home/mastodon/live/vendor/bundle/ruby/3.0.0/bundler/gems/omniauth-cas-4211e6d05941` does not exist. (Bundler::PathError)
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:211:in `load_spec_files'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/path.rb:107:in `local_specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:178:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/lazy_specification.rb:88:in `__materialize__'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:75:in `block in materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `map!'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:468:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:190:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:238:in `specs_for'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/runtime.rb:18:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler.rb:151:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `block in <top (required)>'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:136:in `with_level'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:88:in `silence'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `<top (required)>'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /home/mastodon/live/config/boot.rb:10:in `<top (required)>'
	from /home/mastodon/live/bin/tootctl:4:in `require_relative'
	from /home/mastodon/live/bin/tootctl:4:in `<main>'
/usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:214:in `rescue in load_spec_files': https://github.com/stanhu/omniauth-cas.git (at 4211e6d@4211e6d) is not yet checked out. Run `bundle install` first. (Bundler::GitError)
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:210:in `load_spec_files'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/path.rb:107:in `local_specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:178:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/lazy_specification.rb:88:in `__materialize__'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:75:in `block in materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `map!'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:468:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:190:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:238:in `specs_for'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/runtime.rb:18:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler.rb:151:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `block in <top (required)>'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:136:in `with_level'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:88:in `silence'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `<top (required)>'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /home/mastodon/live/config/boot.rb:10:in `<top (required)>'
	from /home/mastodon/live/bin/tootctl:4:in `require_relative'
	from /home/mastodon/live/bin/tootctl:4:in `<main>'
/usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/path.rb:209:in `load_spec_files': The path `/home/mastodon/live/vendor/bundle/ruby/3.0.0/bundler/gems/omniauth-cas-4211e6d05941` does not exist. (Bundler::PathError)
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:211:in `load_spec_files'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/path.rb:107:in `local_specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/source/git.rb:178:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/lazy_specification.rb:88:in `__materialize__'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:75:in `block in materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `map!'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/spec_set.rb:72:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:468:in `materialize'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:190:in `specs'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/definition.rb:238:in `specs_for'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/runtime.rb:18:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler.rb:151:in `setup'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `block in <top (required)>'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:136:in `with_level'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/ui/shell.rb:88:in `silence'
	from /usr/share/rubygems-integration/all/gems/bundler-2.3.5/lib/bundler/setup.rb:20:in `<top (required)>'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /home/mastodon/live/config/boot.rb:10:in `<top (required)>'
	from /home/mastodon/live/bin/tootctl:4:in `require_relative'
	from /home/mastodon/live/bin/tootctl:4:in `<main>'`

could anyone know why it wont work as cronjob when the user is mastodon?

**Comment by StrangeMinds1960** [comment]
This was my crontab prior to moving Mastodon to a new server, the first few lines are (I suspect) important.  Take what you wish from this

SHELL=/bin/bash
PATH=/home/mastodon/.rbenv/shims:/home/mastodon/.rbenv/bin:/usr/local/bin:/usr/bin:/bin
RAILS_ENV=production
0 0 * * * cd /home/mastodon/live/bin && ./tootctl media remove
30 0 * * * cd /home/mastodon/live/bin && ./tootctl media remove --prune-profiles --include-follows
0 1 * * * cd /home/mastodon/live/bin && ./tootctl media remove-orphans
30 1 * * * cd /home/mastodon/live/bin && ./tootctl accounts cull
0 4 1 * * cd /home/mastodon/live/bin && ./tootctl preview_cards remove --days 30

**Reply by Corrosive1** [reply]
Thanks, just wish fir it to be a cronjob for a single file, wonder where the bug lies.
